juicer/scikit_learn/etl_operation.py

```diff
@@ -955,8 +955,7 @@ def __init__(self, parameters,  named_inputs, named_outputs):
 
         self.weights = float(self.parameters.get('weights', 50))/100
         self.seed = self.parameters.get("seed", 'None')
-        self.seed = 'None' if self.seed == "" else self.seed
-
+        self.seed = 'None' if self.seed == "" and 4294967296 <= int(self.seed) < 0 else self.seed
```

Comment:
I think this you wanted:
self.seed = 'None' if (self.seed == "") or (4294967296 <= int(self.seed) < 0) else self.seed
in this case, use **or** and parentheses for clarity.

Reply:
```
self.seed = self.parameters.get("seed")
self.seed = 'None' if (not self.seed) or (
    4294967296 <= int(self.seed) < 0) else self.seed
```
Notice that I removed the default value 'None' from `self.parameters.get("seed")`

Reply:
Ok, thanks